docs/.prettierrc.yaml

```diff
@@ -0,0 +1 @@
+proseWrap: "always"
```

Comment:
Should we not clearly define the rules here?

Its hard to get a sense otherwise and it would be great to have a way on running the linter locally.

Reply:
About formatting locally: There are two yarn commands analogue to the console npm commands for linting. This is also described in CONTRIBUTING.md as well as in the docs/README.md.

What's missing, however, is a dockerized command for formatting. I'll catch up on that.

Reply:
I'll make the rules explicit 👍